### ISSUE TITLE
Add more inline code to SQL code

### ISSUE THREAD

**Issue by edwardsmarc** [edwardsmarc/PostgreSQL-Table-Translation-Framework#46]
* Comment from Client - end users will not necessarily be power users. 
* They would like some more comments in the code explaining what's going on.

**Comment by edwardsmarc** [comment]
* I could potentially do this as a way to learn the engine functions.

**Comment by pedrogit** [comment]
We can do it when we will review the code together.

**Comment by pedrogit** [comment]
To be closed? Are we happy with the added comments?

**Comment by edwardsmarc** [comment]
No, I haven't addressed this yet. I'll do it before we do the next release.
